src/modules/logger/logged_topics.cpp

```diff
@@ -205,6 +206,7 @@ void LoggedTopics::add_estimator_replay_topics()
 	add_topic("vehicle_status");
 	add_topic("vehicle_visual_odometry");
 	add_topic_multi("distance_sensor");
+	add_topic_multi("battery_status");
```

Comment:
```suggestion
	add_topic_multi("battery_status", 0, 2);
```

This saves a little bit of memory by only trying to log the first 2 battery_status instances at full rate (0 interval).